pymc3/smc/sample_smc.py

```diff
@@ -156,6 +146,29 @@ def sample_smc(
         %282007%29133:7%28816%29>`__
     """
 
+    if kernel is not None:
```

Comment:
How are we going to select if the kernel is metropolis, independent Metropolis-Hasting, HMC, etc?

Reply:
Haven't thought about that yet. We might use the kernel argument for that, in which case I'll remove the DeprecationWarning. This is just temporarily.